### PR TITLE
List styles

### PR DESCRIPTION
**Pull request by Dariasz**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,22 +1,21 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github) {|repo| "https://github.com/#{repo}.git"}
```
Comment (Dariasz) [inline Gemfile]: spaces

File: Gemfile [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
 ruby '2.5.1'
 
 gem 'rails', '~> 5.2.2', '>= 5.2.2.1'
+gem "font-awesome-rails"
```
Comment (Dariasz) [inline Gemfile]: quoutes

File: Gemfile [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 gem 'puma', '~> 3.11'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
-
 gem 'coffee-rails', '~> 4.2'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem "slim"
 gem 'bootsnap', '>= 1.1.0', require: false
-
```
Comment (Dariasz) [inline Gemfile]: restore this line

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
```
Comment (Dariasz) [inline app/assets/stylesheets/application.scss]: add blank line

File: app/assets/stylesheets/lists.scss [hunk 5]
```diff
@@ -1,6 +1,11 @@
 // Place all the styles related to the lists controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@import "font-awesome";
+
+body, html {
```
Comment (Dariasz) [inline app/assets/stylesheets/lists.scss]: set body in application.scss

File: app/assets/stylesheets/lists.scss [hunk 6]
```diff
@@ -58,30 +28,28 @@
   display: flex;
   justify-content: space-between;
   flex-direction: row;
-  padding-bottom: 20px;
-  border-bottom: #730E15 solid 2px;
+  padding-bottom: 1px;
+  border: #730E15 solid 2px;
 }
 
 .inside-container-item {
```
Comment (Dariasz) [inline app/assets/stylesheets/lists.scss]: add BEM convention

File: app/assets/stylesheets/lists.scss [hunk 7]
```diff
@@ -90,6 +58,39 @@ a:hover {
   color: white;
 }
 
+.show-card{
```
Comment (Dariasz) [inline app/assets/stylesheets/lists.scss]: spaces

File: app/controllers/lists_controller.rb [hunk 8]
```diff
@@ -14,19 +14,22 @@ def new
   def edit; end
 
   def create
-    List.create(list_params)
-    redirect_to lists_path
+    @list = List.new(list_params)
```
Comment (Dariasz) [inline app/controllers/lists_controller.rb]: add empty line

Comment (Dariasz) [reply app/controllers/lists_controller.rb]: but what if these actions fail ?

File: app/controllers/tasks_controller.rb [hunk 9]
```diff
@@ -3,72 +3,42 @@
...
-
-    respond_to do |format|
-      if @task.save
-        format.html { redirect_to @task, notice: 'Task was successfully created.' }
-        format.json { render :show, status: :created, location: @task }
-      else
-        format.html { render :new }
-        format.json { render json: @task.errors, status: :unprocessable_entity }
-      end
+    if @task.save
```
Comment (Dariasz) [inline app/controllers/tasks_controller.rb]: same

File: Gemfile (hunk 1 above)
Comment (ajeffika) [reply Gemfile]: done in other branch

File: Gemfile (hunk 2 above)
Comment (ajeffika) [reply Gemfile]: done in other branch

File: Gemfile (hunk 3 above)
Comment (ajeffika) [reply Gemfile]: done in other branch

**Pull request merged**